coordinatorlib/src/main/protobuf/LogData.proto

```diff
@@ -130,5 +130,9 @@ message RowsChangedPreview {
 message SecondaryReindex {
 }
 
+message SecondaryAddIndex {
+  required string fieldName = 2;
```

Comment:
2?  I don't think it matters from a technical point of view, but I'm wondering why you picked 2 to start with.

Reply:
It was copied from somewhere else and i forgot to adjust the index.